reflex/compiler/compiler.py

```diff
@@ -78,11 +78,24 @@ def _compile_page(
         state: The app state.
         connect_error_component: The component to render on sever connection error.
 
+    Raises:
+        ValueError: if a conflict on "tag/alias" is detected for an import.
+
     Returns:
         The compiled component.
     """
     # Merge the default imports with the app-specific imports.
     imports = utils.merge_imports(DEFAULT_IMPORTS, component.get_imports())
+    # Make sure that the same Tag is not used in multiple import.
```

Comment:
Nice check! We could also move this into the `merge_imports` function. Or maybe add a function `utils.validate_imports` to keep this function cleaner